logstash-core/lib/logstash/filter_delegator.rb

```diff
@@ -13,15 +13,14 @@ class FilterDelegator
     ]
     def_delegators :@filter, *DELEGATED_METHODS
 
-    def initialize(logger, klass, metric, *args)
-      options = args.reduce({}, :merge)
-
+    def initialize(logger, klass, metric, plugin_args)
```

Comment:
Renamed options to `plugin_args` to be more clear and consistent across various classes.